compiler/src/dotty/tools/dotc/typer/Inliner.scala

```diff
@@ -252,7 +253,8 @@ object Inliner {
             case tree: Bind => finalize(untpd.Bind(tree.name, transform(tree.body))(curSource))
             case tree: TypeTree => finalize(tpd.TypeTree(tree.tpe))
             case tree: DefTree => super.transform(tree).setDefTree
-            case _ => super.transform(tree)
+            case EmptyTree => tree
+            case _ => fixSpan(super.transform(tree))
```

Comment:
I thought `finalize` will ensure `fixSpan` is called. Just curious if there is something to fix for `TreeCopier`.

Reply:
Probably not. This kind of span/source change is only done here. When we will finally support JSR-45 we will remove all this code anyway.

Reply:
I was thinking about overridding `def postProcess` in `TreeCopier`:

```scala
class Reposition extends TreeMap(cpyWithNewSource) {
  override val cpy: TypedTreeCopier = new TypedTreeCopier() {
      protected def postProcess(tree: Tree, copied: untpd.Tree): copied.ThisTree[T] = ...
      protected def postProcess(tree: Tree, copied: untpd.MemberDef): copied.ThisTree[T] = ...
  }
}
``` 

No need to address in this PR.

Reply:
Maybe. I would not worry to much about this code and push just port the support JSR-45 from Scala 2.